deploy/scripts/src/mobile/utils/k6/k6-utils.js

```diff
@@ -1,2 +1,16 @@
 // # sourceMappingURL=index.js.map
+
+// Copyright 2023 Crown Copyright (Government Digital Service)
```

Comment:
```suggestion
// Copyright 2022 Grafana Labs
```
The Copyright here belongs to Grafana Labs.

Also I would move this notice to the top of the file for visibility. Otherwise it looks good to me 😃

Reply:
Thanks - updated!